library/cartridge_edit_topology.py

```diff
@@ -347,6 +386,11 @@ def get_replicasets_params_for_changing_failover_priority(replicasets, cluster_r
     return replicasets_params, None
 
 
+#########################################################
+# Collect difference between real and inventory servers #
```

Comment:
`servers` -> `instances`. An inventory does not have configured servers. Thi name is used only because of `edit_topology` params names.

Reply:
BTW, I don't like the word `real`. Maybe `current`?